versioned_docs/version-23.10/releases/centreon-os.mdx

```diff
@@ -351,6 +351,21 @@ Release date: `October 30, 2023`
 
 ## Centreon Collect
 
+### 23.10.6
+
+Release date: `April 08, 2024`
+
+<details open>
+  <summary>Bug Fixes</summary>
+
+- [Broker] BA initialization could result in KPIs having incoherent statuses. This was breaking the statuses of the BAs.
+- [Broker] Fixed an issue that caused Broker to crash if an event was dispatched at the same time the process was reloaded.
+- [Collect] Removed the unnecessary check of Anomaly Detection configuration files during the configuration test before deployment.
+- [Collect] Use perl-interpretor instead of perl for centreon-connector-perl packaging (rpm).
+- [Collect] Use perl-base instead of perl for centreon-connector-perl packaging (deb).
```

Comment:
```suggestion
- [Collect] perl-interpretor is now used instead of perl for centreon-connector-perl packaging (rpm).
- [Collect] perl-base is now used instead of perl for centreon-connector-perl packaging (deb).
```